specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/2018-01-01/ResourceHealth.json

```diff
@@ -0,0 +1,882 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-01-01",
+    "title": "Microsoft.ResourceHealth",
+    "description": "The Resource Health Client."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ResourceHealth/events": {
+      "get": {
+        "tags": [
+          "Events"
+        ],
+        "operationId": "Events_ListBySubscriptionId",
+        "description": "Lists current service health events in the subscription. Use the nextLink property in the response to get the next page of events.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/FilterParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The body contains the list of the current service health events in the subscription",
```

Comment:
Nit: Replace `The body contains the` with `The` in this Swagger - since this description makes it into high level artifacts like SDKs and such, descriptions should rather not talk about REST technicalities. 🙂

Reply:
Fixed